snooty/types.py

```diff
@@ -169,6 +172,42 @@ def __load(self) -> None:
             self._checksum = hashlib.blake2b(self._data, digest_size=32).hexdigest()
 
 
+@dataclass
+class TargetDatabase:
```

Comment:
Aha! The TargetDatabase. Does this mean we are covering this ticket as well? https://jira.mongodb.org/browse/DOCSP-6657

Reply:
It just lays a little bit of groundwork for that ticket.